Gemfile

```diff
@@ -25,3 +25,7 @@ end
 group :ci do
   gem 'simplecov', '~> 0.16.0'
 end
+
+if RUBY_ENGINE == 'truffleruby'
+  gem 'haml', '~> 4.0'
```

Comment:
Minor note: it's also possible to use:
```ruby
gem 'haml', '~> 4.0', platforms: [:truffleruby]
```

Reply:
Ah. Nope. That ends up disabling haml for all the other platforms, which isn't what we want.